package.json

```diff
@@ -38,6 +38,7 @@
     "test:codacy-upload": "echo `cat ./coverage/lcov.info | codacy-coverage`",
     "test:coverage": "istanbul check-coverage --statements 80 --lines 80 --functions 80 --branches 80",
     "test:watch": "karma start --auto-watch --no-single-run",
+    "test:audit": "./bin/run-yarn-audit.sh -f yarn-audit-known-issues",
```

Comment:
you've to call this from Travis file

Reply:
I second that. Needs to be in travis file too.

Reply:
Sure. I just missed that.